### PR TITLE
Add compression to the data stored in DB

### PR DESCRIPTION
**Pull request by siy**

### CODE REVIEW

File: radixdlt/src/main/java/com/radixdlt/store/berkeley/BerkeleyLedgerEntryStore.java [hunk 1]
```diff
@@ -280,14 +278,25 @@ public void commit(AID aid) {
 		}
 	}
 
+	private LedgerEntryStoreResult doStore(
+			byte prefix,
+			LedgerEntry ledgerEntry,
+			LedgerEntryIndices indices,
+			Transaction transaction) throws IOException {
+		byte[] ledgerEntryData = encode(ledgerEntry);
+		log.debug("Storing atom {} with size {}", ledgerEntry.getAID(), ledgerEntryData.length);
```
Comment (clstrfsck) [inline radixdlt/src/main/java/com/radixdlt/store/berkeley/BerkeleyLedgerEntryStore.java]: Probably should be trace level, rather than debug.

**Review by clstrfsck — APPROVED** [review]
Looks good.  Minor suggestion about log level.

File: radixdlt/src/main/java/com/radixdlt/store/berkeley/BerkeleyLedgerEntryStore.java (hunk 1 above)
Comment (siy) [reply radixdlt/src/main/java/com/radixdlt/store/berkeley/BerkeleyLedgerEntryStore.java]: Done

**Review by LukasGasior1 — APPROVED** [review]
LGTM

**Pull request merged**